src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs

```diff
@@ -468,7 +468,7 @@ private async Task HandleMessageAsync(Message message, CancellationToken cancell
                 Logger.LogTrace("No operation ID was found on the message");
             }
 
-            MessageCorrelationInfo correlationInfo = message.GetCorrelationInfo();
+            MessageCorrelationInfo correlationInfo = message.GetCorrelationInfo(Settings.Options.Correlation.TransactionIdPropertyName);
```

Comment:
What if it's null? Are we sure this is safe?

Reply:
It's should be checked when creating the options, but we can always add some paranoia checks if you like :)